### DISCUSSION TITLE
Behind the Firewall: Checking into the Code Security Community 🤖🪐

### DISCUSSION THREAD

**Discussion by ghostinhershell** [community/community#142537] [Code Security]
Hey hey, @ghostinhershell here reporting in for with a Code Security Community Check-In 🗞

We’ve got some community shout-outs to give, questions that need community expertise to answer, updates on what’s new in the Code Security space and updates on GitHub Universe 2024 🪐

## 🤖 Code Security Shoutouts
- @ThisIs-Developer (check out their project spotlight [here](https://github.com/orgs/community/discussions/142422)) lent community members a hand with an answer to the post [Could not get my account lmysticl](https://github.com/orgs/community/discussions/140119).
- Shout out to @MuxN4 for teaching us [how floating-point numbers are represented in binary](https://github.com/orgs/community/discussions/141920)
- A self-serve solution is always a win ✨shoutout to @Dev202409 for finding a solution to their post [Account A signed in everywhere, commit is assigned to account B. How to prevent this?](https://github.com/orgs/community/discussions/140769). 

--
**Have you been helped by a community superstar that you’d like to see celebrated? Tag them to nominate them below!**

For community members who post questions, [marking responses that solves your question as the answer](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/participating-in-a-discussion#marking-a-comment-as-an-answer) goes a long way. When you mark a comment in the Discussion as an answer, GitHub will highlight the comment to help other community members quickly find the answer as well.

<img src="https://github.com/user-attachments/assets/b89e7161-c2d1-47e2-a058-242df633e6b2" alt="The image is a screenshot showing where users can click to mark a response as the answer" width="800" height="300">

Remember to thank these and other community members that take the time to comment and help troubleshoot these problems. An acknowledgement of a job well done can have an incredible effect and make someone’s day. ☀️ We’re here to learn, discover and connect so try and mark a comment as an answer if suitable, give a thank you and select a reaction as a way to show your appreciation! 

## 🧠 Help Wanted! Questions that Need Answers
Got your thinking cap on? These popular community posts could use your expertise!
- [Unable to Push to GitHub Due to Secret Scanning](https://github.com/orgs/community/discussions/140034)
- [Need help updating rollup to fix DOM Clobbering vulnerability :(](https://github.com/orgs/community/discussions/139775)
- [How Do You Handle Data Privacy and Security in Hybrid AI Systems?](https://github.com/orgs/community/discussions/139176)

Looking to uplevel your own security knowledge so you can tackle some of these popular questions yourself? Check out our [GitHub Advanced Security certification](https://resources.github.com/learn/certifications/)! We even have a [prep course in the community](https://github.com/orgs/community/discussions/137197) to help you study ✨ 

## 🆕 What’s New in Code Security?
A number of feedback posts have been published in the community for recent releases in the GitHub Security World. Have you seen them? 👀 
- [Secret scanning: on-demand revocation for GitHub personal access tokens - feedback](https://github.com/orgs/community/discussions/139967)
- [Secret scanning: public leak locations and alert de-duplication across an organization or enterprise - feedback](https://github.com/orgs/community/discussions/141497)
- [Secret Scanning & Security Dashboard Updates 📢](https://github.com/orgs/community/discussions/139243)
- [EPSS Scores in the GitHub Advisory Database 🚀](https://github.com/orgs/community/discussions/141713)
- [CodeQL 2.19.0: TypeScript 5.6 and Go 1.23 support, new queries for JavaScript and Ruby](https://github.com/orgs/community/discussions/141715)
- [Announcing TISAX for GitHub 🚀](https://github.com/orgs/community/discussions/141711)

Please check them out and, if you have any feedback to share, sound off in the comments section of the applicable post.

## 🪐✨ Universe 2024
GitHub Universe 2024 will be an unparalleled gathering of the brightest minds, companies, and innovators in the industry. Plus this year is the 10 year anniversary of the event 🎉 Will you be there? 👀 
- **LAST CALL to Get Tickets:** Vist the [Universe website](https://githubuniverse.com/) to purchase your tickets and plan your agenda!  

Excited for Universe 2024? Keep the conversation going year-round with GitHub's online community—your go-to space for insights, networking, and the latest in all things GitHub. Join in to connect with peers, access exclusive resources, and keep building together before, during, and after the event!
- **Get Started with Universe 2024:** With less than a week to go, join us in everything Universe in the community. [Check out this Community Post](https://github.com/orgs/community/discussions/141919) to start the celebration!
- **Open Source**
  - [Open source for all: Skilling the next generation of developers](https://github.com/orgs/community/discussions/141485)
- **GHAS / Security / Code Security**
  - [Go big! Efficiently deploy and customize security tooling at enterprise scale](https://github.com/orgs/community/discussions/141479)
  - [Enterprise code security 101: Getting started with GitHub Advanced Security](https://github.com/orgs/community/discussions/139507) (Universe 2024 Workshop 🪐 )
  - [Why (and how) IKEA, WirelessCar, and SAP adopted GitHub Advanced Security](https://github.com/orgs/community/discussions/141572)
  - [Tackling supply chain vulnerabilities with Dependabot [SEC1968D]](https://github.com/orgs/community/discussions/141502)
  - [Real talk on navigating code security in the age of AI](https://github.com/orgs/community/discussions/141704)

## ✍️ Quote of the Month
"Any problem can be solved by investing more heavily into it, but there’s always a more efficient solution. If you’re convinced that you must surge investment, check if your mental model is leading you astray."
> From _[Move Past Incident Response to Reliability](https://github.com/readme/guides/incident-response)_ by Will Larson // CTO, Calm

—--
A very big thank you to everyone in the community who actively contributes thoughts and answers to the questions that come into the New to GitHub category!

**Comment by Rock1raw** [comment]
How can I get everything 

organized?
